src/imgtools/dicom/input/sr_utils.py

```
@@ -0,0 +1,94 @@
+    sr: DicomInput,
+) -> tuple[SR_RefSeries, SR_RefSOPs] | None:
+    """Get the `ReferencedSeriesInstanceUID`s from an SR file
+
+    SR Dicom files can reference multiple SeriesInstanceUIDs and many SOPInstanceUIDs
+
+    Since we might need to match on SOP Instance UIDs if the reference is
+    a MR, we also get the SOP Instance UIDs
+
+    """
+    sr = load_dicom(sr)
+    if not "CurrentRequestedProcedureEvidenceSequence" in sr:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)